bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,290 @@
+# Babylon Validator Setup
+
+## Table of Contents
+
+1. [System Requirements](#system-requirements)
+2. [Install Babylon Binary](#install-babylon-binary)
+3. [Setup Node Home Directory and Configure](#setup-your-node-home-directory-and-configure)
+4. [Setup Required Keys](#setup-the-required-keys-for-operating-a-validator)
+5. [Sync Node](#sync-node)
+6. [Get Funds](#get-funds)
+7. [Create Validator](#create-validator)
+
+## System Requirements
+
+The Babylon node is built using the Cosmos SDK and has similar system requirements with typical Cosmos SDK ecosystem nodes.
+
+As a reference, the following system specification is typically used by validators and found to be comfortable for high up-time operations.
+
+- Quad Core or larger AMD or Intel (amd64) CPU
+- 32GB RAM
+- 1TB NVMe Storage
```

Comment:
From the node benchmark, we estimate that we require 2TB to store 1M BTC delegations.
I suggest that we adjust this requirement accordingly to 2TB of storage, at least.